ps2/hangmanlayout.txt

```diff
@@ -1,9 +1,76 @@
-warnings = 3
-input = 
-if input not alpha and warnings > 0:
-    warnings -= 1    
+    guesses_remaining = 6
```

Comment:
What is this file?

Reply:
It's to scaffold the code before I write it.

Reply:
?

Reply:
no

Reply:
lol

Reply:
I guess you just keep everything in your head.